### PR TITLE
D2.2 feedback issues resolution

### PR DESCRIPTION
**Pull request by schivmeister**
Output/reports will be re-generated and committed separately.

### CODE REVIEW

File: mappings/package_cn_v1.3/transformation/mappings/ChangeInformation_v1.3-1.6.rml.ttl [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+tedm:MG-ChangeInformation_ND-Change a rr:TriplesMap ;
+    rr:predicateObjectMap
+        [
+            rdfs:label "BT-13716-notice (object)";
+            rr:predicate epo:relatesToEFormSectionIdentifier ;
+            rr:objectMap
+                [
+                    tedm:minSDKVersion "1.3" ;
+                    tedm:minSDKVersion "1.6" ;
```
Comment (cristianvasquez) [inline mappings/package_cn_v1.3/transformation/mappings/ChangeInformation_v1.3-1.6.rml.ttl]: I see two tedm:minSDKVersion specified, shouldn't the second one be maxSDKVersion or something like that?

File: mappings/package_cn_v1.3/transformation/mappings/ChangeInformation_v1.3-1.6.rml.ttl [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        ] ;
+    rr:predicateObjectMap
+        [
+            rdfs:label "BT-13716-notice (value)";
+            rdfs:comment "Change Notice Section Identifier of MG-Identifier under ND-Changed";
+            rr:predicate skos:notation  ;
+            rr:objectMap
+                [
+                    tedm:minSDKVersion "1.3" ;
+                    tedm:minSDKVersion "1.6" ;
```
Comment (cristianvasquez) [inline mappings/package_cn_v1.3/transformation/mappings/ChangeInformation_v1.3-1.6.rml.ttl]: same as above

File: mappings/package_cn_v1.3/transformation/mappings/ChangeInformation_v1.3-1.6.rml.ttl (hunk 1 above)
Comment (schivmeister) [reply mappings/package_cn_v1.3/transformation/mappings/ChangeInformation_v1.3-1.6.rml.ttl]: thanks for spotting, fixed

File: mappings/package_cn_v1.4/transformation/mappings/ChangeInformation_v1.3-1.6.rml.ttl [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+tedm:MG-ChangeInformation_ND-Change a rr:TriplesMap ;
+    rr:predicateObjectMap
+        [
+            rdfs:label "BT-13716-notice (object)";
+            rr:predicate epo:relatesToEFormSectionIdentifier ;
+            rr:objectMap
+                [
+                    tedm:minSDKVersion "1.3" ;
+                    tedm:minSDKVersion "1.6" ;
```
Comment (cristianvasquez) [inline mappings/package_cn_v1.4/transformation/mappings/ChangeInformation_v1.3-1.6.rml.ttl]: tedm:maxSDKVersion?

File: mappings/package_cn_v1.3/transformation/mappings/ChangeInformation_v1.3-1.6.rml.ttl (hunk 2 above)
Comment (schivmeister) [reply mappings/package_cn_v1.3/transformation/mappings/ChangeInformation_v1.3-1.6.rml.ttl]: thanks for spotting, fixed

File: mappings/package_cn_v1.4/transformation/mappings/ChangeInformation_v1.3-1.6.rml.ttl [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+        ] ;
+    rr:predicateObjectMap
+        [
+            rdfs:label "BT-13716-notice (value)";
+            rdfs:comment "Change Notice Section Identifier of MG-Identifier under ND-Changed";
+            rr:predicate skos:notation  ;
+            rr:objectMap
+                [
+                    tedm:minSDKVersion "1.3" ;
+                    tedm:minSDKVersion "1.6" ;
```
Comment (cristianvasquez) [inline mappings/package_cn_v1.4/transformation/mappings/ChangeInformation_v1.3-1.6.rml.ttl]: same as above.

Comment (schivmeister) [reply mappings/package_cn_v1.4/transformation/mappings/ChangeInformation_v1.3-1.6.rml.ttl]: thanks for spotting, fixed

File: mappings/package_cn_v1.4/transformation/mappings/ChangeInformation_v1.3-1.6.rml.ttl (hunk 3 above)
Comment (schivmeister) [reply mappings/package_cn_v1.4/transformation/mappings/ChangeInformation_v1.3-1.6.rml.ttl]: thanks for spotting, fixed

File: mappings/package_cn_v1.5/transformation/mappings/ChangeInformation_v1.3-1.6.rml.ttl [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+tedm:MG-ChangeInformation_ND-Change a rr:TriplesMap ;
+    rr:predicateObjectMap
+        [
+            rdfs:label "BT-13716-notice (object)";
+            rr:predicate epo:relatesToEFormSectionIdentifier ;
+            rr:objectMap
+                [
+                    tedm:minSDKVersion "1.3" ;
+                    tedm:minSDKVersion "1.6" ;
```
Comment (cristianvasquez) [inline mappings/package_cn_v1.5/transformation/mappings/ChangeInformation_v1.3-1.6.rml.ttl]: tedm:maxSDKVersion?

File: mappings/package_cn_v1.5/transformation/mappings/ChangeInformation_v1.3-1.6.rml.ttl [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+        ] ;
+    rr:predicateObjectMap
+        [
+            rdfs:label "BT-13716-notice (value)";
+            rdfs:comment "Change Notice Section Identifier of MG-Identifier under ND-Changed";
+            rr:predicate skos:notation  ;
+            rr:objectMap
+                [
+                    tedm:minSDKVersion "1.3" ;
+                    tedm:minSDKVersion "1.6" ;
```
Comment (cristianvasquez) [inline mappings/package_cn_v1.5/transformation/mappings/ChangeInformation_v1.3-1.6.rml.ttl]: tedm:maxSDKVersion?

File: mappings/package_cn_v1.5/transformation/mappings/ChangeInformation_v1.3-1.6.rml.ttl (hunk 5 above)
Comment (schivmeister) [reply mappings/package_cn_v1.5/transformation/mappings/ChangeInformation_v1.3-1.6.rml.ttl]: thanks for spotting, fixed

File: mappings/package_cn_v1.5/transformation/mappings/ChangeInformation_v1.3-1.6.rml.ttl (hunk 6 above)
Comment (schivmeister) [reply mappings/package_cn_v1.5/transformation/mappings/ChangeInformation_v1.3-1.6.rml.ttl]: thanks for spotting, fixed

**Review by cristianvasquez — CHANGES_REQUESTED** [review]
It looks good overall, but I did notice a minor issue with the metadata. Specifically, there are instances of the following entries:

    tedm:minSDKVersion "1.3";
    tedm:minSDKVersion "1.6";

**Comment by schivmeister** [comment]
> It looks good overall, but I did notice a minor issue with the metadata. Specifically, there are instances of the following entries:
> 
> ```
> tedm:minSDKVersion "1.3";
> tedm:minSDKVersion "1.6";
> ```

indeed, these errors have been fixed. they occurred for all ChangeInformation in the range 1.3-1.6.

**Comment by schivmeister** [comment]
> > It looks good overall, but I did notice a minor issue with the metadata. Specifically, there are instances of the following entries:
> > ```
> > tedm:minSDKVersion "1.3";
> > tedm:minSDKVersion "1.6";
> > ```
> 
> indeed, these errors have been fixed. they occurred for all ChangeInformation in the range 1.3-1.6.

Unfortunately the latest commit and tag undid this fix as the packages slated for output slipped past with the older files (there was a gap of 15-30 minutes with this change) which got hidden among the enormous diff stat. However, as this is a non-critical and non-functional change, we will leave it for now.

**Pull request merged**